core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review.directive.html

```diff
@@ -10,6 +10,9 @@ <h3>Review translation</h3>
 </div>
 
 <div class="modal-body">
+  <div class="oppia-suggestion-review-edit-button-container">
+    <button class="btn btn-secondary" ng-if="!startedEditing && status === 'review'" ng-click="editSuggestion()">Edit</button>
+  </div>
```

Comment:
From the screenshot, it looks like the edit button is on its own line—I meant can we place it inline with the "Translation:" text so that it's directly above the translation box?

Reply:
Done

![translation-suggestion-editing-mode](https://user-images.githubusercontent.com/41409951/114110167-7f4b1000-98f4-11eb-9cf6-aafb0e0f96ee.png)